### ISSUE TITLE
[nc 15.0.0] accessibility in Menue not in transifex

### ISSUE THREAD

**Issue by JSoko** [nextcloud/server#13034]
### Steps to reproduce
1. Accessibility in the Menue should be translatable in transifex
2. https://www.transifex.com/nextcloud/nextcloud/translate/#de/$/151372581?q=text%3Aaccessibility
![grafik](https://user-images.githubusercontent.com/5583290/49877284-5ea64480-fe25-11e8-8f3f-b60b9a7de8fb.png)

### Expected behaviour
Should be shown in tranisfex, so that it can be translated

### Server configuration
**Nextcloud version:** (see Nextcloud admin page)
15.0.0
**Updated from an older Nextcloud/ownCloud or fresh install:**
updated

**Comment by JSoko** [comment]
Dupe of
https://github.com/nextcloud/server/issues/11803
